core/kernel/shared/load.rb

```diff
@@ -154,6 +154,36 @@
     end
   end
 
+  describe "when passed a module for 'wrap'" do
+    ruby_version_is ""..."3.1" do
+      it "does not set the enclosing scope to the supplied module" do
+        path = File.expand_path "wrap_fixture.rb", CODE_LOADING_DIR
+        mod = Module.new
+        @object.load(path, mod)
+
+        Object.const_defined?(:LoadSpecWrap).should be_false
+        mod.const_defined?(:LoadSpecWrap).should be_false
+
+        wrap_module = ScratchPad.recorded[1]
+        wrap_module.should be_an_instance_of(Module)
+      end
+    end
```

Comment:
Could you remove this? It's not really meaningful because before 3.1 the 2nd arg to `load` is just a boolean.